sign-and-combine-ecdsa/nodejs/src/index.ts

```diff
@@ -0,0 +1,186 @@
+import * as ethers from "ethers";
+import { LitContracts } from "@lit-protocol/contracts-sdk";
+import { LitNetwork, LIT_RPC, LIT_CHAINS } from "@lit-protocol/constants";
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import {
+  LitAbility,
+  LitActionResource,
+  LitPKPResource,
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+} from "@lit-protocol/auth-helpers";
+
+import { getEnv } from "./utils";
+import { litActionCode } from "./litAction";
+
+const ETHEREUM_PRIVATE_KEY = getEnv("ETHEREUM_PRIVATE_KEY");
+let capacityTokenId = getEnv("CAPACITY_TOKEN_ID");
```

Comment:
```suggestion
const LIT_CAPACITY_CREDIT_TOKEN_ID = getEnv("LIT_CAPACITY_CREDIT_TOKEN_ID");
```
Constants should always be `const`